hw/dv/dpi/dmidpi/README.md

```diff
@@ -0,0 +1,14 @@
+DMI DPI module for OpenOCD remote_bitbang driver
+================================================
+
+This DPI module provides a "virtual" JTAG connection between a simulated chip
+and [OpenOCD](http://openocd.org/). It makes use of the `remote_bitbang` JTAG
+driver shipped with OpenOCD, which forwards JTAG requests over TCP to a remote
+server. The `dmidpi` module is instantiated in the hardware simulation to
+receive the JTAG requests from OpenOCD and drive DMI pins directly.
+
+Note that this module effectively replaces the TAP inside the debug module.
```

Comment:
In debug spec language, dmidpi is a debug transport module (DTM). It replaces the JTAG DTM with an "OpenOCD remote bitbang DTM" (better name needed). Can you somehow add this information to the README?